components/camera/videosource/server_test.go

```diff
@@ -159,12 +159,12 @@ func TestDualServerSource(t *testing.T) {
 	defer svr.Close()
 	// intrinsics for the image
 	intrinsics := &transform.PinholeCameraIntrinsics{
-		Width:  1024,
-		Height: 768,
+		Width:  128,
+		Height: 72,
 		Fx:     821.32642889,
 		Fy:     821.68607359,
-		Ppx:    494.95941428,
-		Ppy:    370.70529534,
+		Ppx:    49.495941428,
+		Ppy:    37.070529534,
```

Comment:
LGTM as long as all the tests pass. I'm mostly worried about this one since the resolution decrease wasn't perfectly 10x and the PPx/PPy reduction was.

Reply:
The projection doesn't have to be perfect for the test (if this would make the resulting pointcloud unaligned, that is OK - just wanted to make sure it didn't crash or anything).